Removing unused allow list arguments from propagate_qconfig and helper

HDCharles opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #71104

Summary: This shouldn't change any functionality given that those
variables were not used. It should be noted that a similar variable is
used in add_observer which is why it wasn't removed from there.

Test Plan: python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33510352](https://our.internmc.facebook.com/intern/diff/D33510352)

HDCharles committed:
Removing unused allow list arguments from propagate_qconfig and helper

Summary: This shouldn't change any functionality given that those
variables were not used. It should be noted that a similar variable is
used in add_observer which is why it wasn't removed from there.

Test Plan: python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33510352).

HDCharles committed:
Update on "Removing unused allow list arguments from propagate_qconfig and helper"


Summary: This shouldn't change any functionality given that those
variables were not used. It should be noted that a similar variable is
used in add_observer which is why it wasn't removed from there.

Test Plan: python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33510352](https://our.internmc.facebook.com/intern/diff/D33510352)

[ghstack-poisoned]

HDCharles commented:
@HDCharles has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33510352).

HDCharles committed:
Update on "Removing unused allow list arguments from propagate_qconfig and helper"


Summary: This shouldn't change any functionality given that those
variables were not used. It should be noted that a similar variable is
used in add_observer which is why it wasn't removed from there.

Test Plan: python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33510352](https://our.internmc.facebook.com/intern/diff/D33510352)

[ghstack-poisoned]

HDCharles committed:
Update on "Removing unused allow list arguments from propagate_qconfig and helper"


Summary: This shouldn't change any functionality given that those
variables were not used. It should be noted that a similar variable is
used in add_observer which is why it wasn't removed from there.

Test Plan: python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33510352](https://our.internmc.facebook.com/intern/diff/D33510352)

[ghstack-poisoned]

HDCharles committed:
Update on "Removing unused allow list arguments from propagate_qconfig and helper"


Summary: This shouldn't change any functionality given that those
variables were not used. It should be noted that a similar variable is
used in add_observer which is why it wasn't removed from there.

Test Plan: python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33510352](https://our.internmc.facebook.com/intern/diff/D33510352)

[ghstack-poisoned]

vkuzo reviewed (approved):
technically this is bc-breaking since we are removing an argument to `prepare_qat`, although practically it should be little to no impact since the argument doesn't do anything.  We can call this out in the release notes.

thanks for fixing this!

HDCharles committed:
Update on "Removing unused allow list arguments from propagate_qconfig and helper"


Summary: This shouldn't change any functionality given that those
variables were not used. It should be noted that a similar variable is
used in add_observer which is why it wasn't removed from there.

Test Plan: python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33510352](https://our.internmc.facebook.com/intern/diff/D33510352)

[ghstack-poisoned]

HDCharles committed:
Update on "Removing unused allow list arguments from propagate_qconfig and helper"


Summary: This shouldn't change any functionality given that those
variables were not used. It should be noted that a similar variable is
used in add_observer which is why it wasn't removed from there.

Test Plan: python test/test_quantization.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D33510352](https://our.internmc.facebook.com/intern/diff/D33510352)

[ghstack-poisoned]
